MobiFlightConnector.csproj

```diff
@@ -1139,7 +1139,7 @@
       <CopyToOutputDirectory>Always</CopyToOutputDirectory>
     </None>
     <None Include="Presets\xplane_hubhop_presets.json">
-      <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
+      <CopyToOutputDirectory>Always</CopyToOutputDirectory>

```

Comment:
I changed this for testing, i made the presets files older and I wanted to make sure they will get copied over always.

Reply:
> I changed this for testing, i made the presets files older and I wanted to make sure they will get copied over always.

see my first comment